buildSrc/src/main/java/Config.kt

```diff
@@ -1,10 +1,10 @@
 import java.math.BigDecimal
 
 object Config {
-    val kotlinVersion = "1.4.10"
+    val kotlinVersion = "1.4.30"
     val kotlinStdLib = "stdlib-jdk8"
 
-    val springBootVersion = "2.3.4.RELEASE"
+    val springBootVersion = "2.4.4"
```

Comment:
this is necessary to be compatible with dokka and spring dependency management.

https://github.com/Kotlin/dokka/issues/1791

spring still compiles with API 1.3 compatibility though, so it should not be a biggie, @maciejwalkowiak to confirm

cc @bruno-garcia @maciejwalkowiak